frontend/frontend.py

```diff
@@ -10,4 +10,5 @@ def __init__(self):
     def start(self):
         while True:
             self.dashboard.update()
+            self.dashboard.update_widget("ControllerWidget")
```

Comment:
`p2s/manipulator` module will call this command, so we can delete this line.

Reply:
o k